MINOR increasing queued.max.requests for production recommendations in 0.9 and newer.

alexlod opened the pull request.

ijuma commented:
Thanks for the PR. This change seems fine. For 0.11.0 and newer, we did a bit more:

https://github.com/apache/kafka/commit/be98865dfcddd38e3e411d41126fa6049c509cae#diff-22956c3cc0f5bb53d95fb30c9776f81d
https://github.com/apache/kafka/commit/8bd8751aa776990d1847e4d6ab187a0aea90c051#diff-22956c3cc0f5bb53d95fb30c9776f81d

Do we want to do the same here?

alexlod commented:
Good idea @ijuma. I forgot about that PR from way back then :). I updated this PR to be inline with what we changed back then.

Also, the change starting with `<li>ZooKeeper` removes a trailing space. I'm happy to get rid of that change if you'd like.

Thanks again!

mjsax commented:
Closing this PR as abandoned. (We can reopen if necessary.)
